std_msgs/Header header
cinempc/DroneAndCameraState drone_state
float32[] coeffs
cinempc/TargetStates[] targets
float32 floor_pos
Constraints constraints
